Source/Eto.Gtk/Forms/Controls/DropDownHandler.cs

```diff
@@ -64,6 +64,18 @@ public virtual void HandleChanged(object sender, EventArgs e)
 			}
 		}
 
+		public override Size Size
+		{
+			get { return base.Size; }
+			set
+			{
+				if (value.Width != Size.Width)
+					text.Ellipsize = Pango.EllipsizeMode.End;
```

Comment:
I did not know that you used `-1` for setting auto size... I'll change the code as soon as I can.